### PR TITLE
feat(signup page): update design

### PR DESCRIPTION
**Pull request by andreybakanovsky**

### CODE REVIEW

**Comment by andreybakanovsky** [comment]
image

File: app/views/devise/registrations/new.html.slim [hunk 1]
```diff
@@ -1,29 +1,55 @@
...
+              | Login &nbsp;
+              figure.image.is-16x16
+                = image_tag('arrow-right.svg', alt: '->')
+          br
+          .is-flex.is-justify-content-center
+            | Already have an account? &nbsp;
+            = link_to 'Login now', new_user_session_path, class: 'has-text-weight-bold has-text-budgeting-kid-color'
+  .for-desktop.column.has-background-image.is-flex.is-justify-content-center.is-align-items-center
+    figure.image.is-coins
+      = image_tag('coins.svg')
```
Comment (VladislavSokov) [inline app/views/devise/registrations/new.html.slim:55]: add empty line, please

Comment (ka8725) [reply app/views/devise/registrations/new.html.slim:55]: remove?

File: app/views/devise/sessions/new.html.slim [hunk 2]
```diff
@@ -1,30 +1,55 @@
...
+        h2.title.is-3.has-text-centered.font-frank-ruhl-libre Login to your account
+        - if Devise.omniauth_providers.include?(:google_oauth2)
+          .field
+            .buttons.is-centered
+              = link_to user_google_oauth2_omniauth_authorize_path, method: :post, class: 'button has-fixed-width is-outlined is-budgeting-kid-hoverable' do
+                span
+                  figure.image.is-24x24
+                    = image_tag('google.svg', class:'is-rounded',  alt: 'G')
+                | &nbsp; Google
+              /= link_to "#", method: :post, class: 'button has-fixed-width is-outlined is-budgeting-kid-hoverable' do
```
Comment (ka8725) [inline app/views/devise/sessions/new.html.slim:15]: do we need this?

**Review by ka8725 — APPROVED** [review]

**Pull request merged**